src/bench/bench.cpp

```diff
@@ -41,15 +41,37 @@ void GenerateTemplateResults(const std::vector<ankerl::nanobench::Result>& bench
 
 } // namespace
 
+std::map<std::string, benchmark::PriorityLevel> map_priority_level = {
+    std::make_pair("high", benchmark::PriorityLevel::HIGH),
+    std::make_pair("medium", benchmark::PriorityLevel::MEDIUM),
+    std::make_pair("low", benchmark::PriorityLevel::LOW),
+    std::make_pair("all", benchmark::PriorityLevel::ALL)
+};
+
+benchmark::PriorityLevel benchmark::StringToPriority(const std::string& str)
+{
+    auto it = map_priority_level.find(str);
+    if (it == map_priority_level.end()) throw std::runtime_error(strprintf("Unknown priority level %s", str));
+    return it->second;
+}
+
+std::string benchmark::PriorityToString(PriorityLevel in_level)
```

Comment:
Currently, there's a bit of inconsistency in the namespace specification (`benchmark::PriorityLevel` in other places). What do you think about putting all the `benchmark` implementations here in `namespace benchmark` and remove all the `benchmark::` specifiers? This makes the code more concise and explicit, and mirrors the header file. (thank you @hebasto for the idea).

Reply:
sure, I thought about it as well but didn't want to expand this further.
Now that it's not just me, will add a small commit for it at the beginning :).